.eslintrc

```diff
@@ -1,176 +1,176 @@
 {
```

Comment:
Yeah, read them all and support 100%. I use them only because of React's new syntax for components.